pkg/whitesource/configHelper.go

```diff
@@ -27,9 +27,12 @@ const configFileName = "wss-unified-agent.config"
 // ConfigOptions contains a list of config options (ConfigOption)
 type ConfigOptions []ConfigOption
 
+// Needed parameters from UAConfiguration to print in verbose=true
+var parametersForLog = []string{"docker.excludeBaseImage", "docker.dockerfilePath"}
```

Comment:
Why do we need to print only these parameters? Maybe it would be helpful to print the whole wss-unified-agent.config?

Reply:
By default, the unified-agent will display all parameters except the ones listed below. Therefore, support has requested to display these as well.